server/tests/integration/customers.test.ts

```diff
@@ -20,15 +20,14 @@ import app from '../../src';
 import customerFixtures from '../fixtures/customers';
 
 describe('Customers endpoints', () => {
-  test('it should fetch a single customer', async () => {
-    const expectedCustomerData = customerFixtures.data?.[0];
-    const customerId = customerFixtures.ids?.[0];
+  describe('GET /customers', () => {
+    test('it should fetch a single customer', async () => {
+      const expectedCustomerData = customerFixtures.responseData?.[0];
+      const customerId = customerFixtures.ids?.[0];
```

Comment:
Why not this instead?
```suggestion
      const customerId = expectedCustomerData.id;
```
I think this can better ensure that the `id` is the correct id of the `expectedCustomerData`, and it is also immediately clear to the reader that the id belongs to the correct data.

Reply:
Oh yea good catch! I was changing the customer one to match and didn't realise I should change that line too 😆